demo-certs/create_certs.sh

```diff
@@ -100,3 +116,4 @@ create_cert kura-adapter $KURA_ADAPTER_KEY_STORE $KURA_ADAPTER_KEY_STORE_PWD
 create_cert artemis $ARTEMIS_KEY_STORE $ARTEMIS_KEY_STORE_PWD
 create_cert coap-adapter $COAP_ADAPTER_KEY_STORE $COAP_ADAPTER_KEY_STORE_PWD
 
+create_client_cert 4711
```

Comment:
missing EOL
this should actually be detected by the checkstyle rules ...

Reply:
Dammit. Looks like the Plugin checks only Java files. I'll check what I can do.